addons/armaos/functions/fnc_terminal_updateOutput.sqf

```diff
@@ -0,0 +1,13 @@
+params ["_computer", "_outputControl"];
+
+[_computer] call AE3_armaos_fnc_terminal_updateBufferVisable;
+
+private _terminal = _computer getVariable "AE3_terminal";
+
+private _terminalBuffer = _terminal get "AE3_terminalBuffer";
+private _terminalBufferVisable = _terminal get "AE3_terminalBufferVisable";
+
+hint format ["BUFFER: \n ***%1*** \n\n BUFFER VISABLE: \n ***%2***", _terminalBuffer, _terminalBufferVisable];
```

Comment:
Debug msg

Reply:
removed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/bdeec0439c3a89ef1583c3b663196a37a6c51e0b